src/main/java/seedu/address/logic/commands/UnassignProjectCommand.java

```diff
@@ -0,0 +1,114 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMPLOYEE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PROJECT;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_EMPLOYEES;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.employee.Employee;
+import seedu.address.model.project.Project;
+
+/**
+ * Unassigns a project from an employee.
+ */
+public class UnassignProjectCommand extends Command {
+
+    public static final String COMMAND_WORD = "unassignP";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Unassigns the project of the employees identified "
+            + "by the index number used in the last employee listing. "
+            + "\n"
+            + "Parameters: PROJECT_INDEX, EMPLOYEE_INDEX (must be a positive integer) "
+            + "pr/[PROJECT_INDEX] em/[EMPLOYEE_INDEX ...]\n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_PROJECT + "1 "
+            + PREFIX_EMPLOYEE + "2 3 1";
+
+    public static final String MESSAGE_UNASSIGN_PROJECT_SUCCESS = "Member(s) have been un-assigned!\n%1$s";
+    public static final String MESSAGE_UNASSIGN_PROJECT_FAILURE = "The following employee is not in the specified "
+            + "project. \n"
+            + "Employee: %1$s (Index: %2$s) "
+            + "Project: %3$s (Index: %4$s)\n"
+            + "Please check the project and employee index again. "
+            + "Using the findP command to isolate the project and it's employees may be useful.";
+
+    private final Index projectIndex;
+    private final List<Index> employeeIndexes;
+
+    /**
+     * @param projectIndex index of the project in the filtered project list to update
+     * @param employeeIndexes indexes of the employees to be unassigned from the project
+     */
+    public UnassignProjectCommand(Index projectIndex, List<Index> employeeIndexes) {
+        requireAllNonNull(projectIndex, employeeIndexes);
+
+        this.projectIndex = projectIndex;
+        this.employeeIndexes = employeeIndexes;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        List<Employee> lastShownEmployeeList = model.getFilteredEmployeeList();
+        List<Project> lastShownProjectList = model.getFilteredProjectList();
+
+        if (projectIndex.getZeroBased() >= lastShownProjectList.size()) {
```

Comment:
There is a special sub-case of the case where the `projectIndex` exceeds the size of the list of projects. Do you think we could separate the case for where no projects exist yet from this general case and give a more specific error message? 

Example:

> "You don't currently have any projects to unassign employees from! Try first creating a project with the addP command"

Nonetheless, this condition does account for that case so test coverage wouldn't be a concern.

Reply:
It's possible I think.